### PR TITLE
[Documentation] Customer and Order Admin Menus

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | no |
| New docs?    | yes |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

File: docs/customization/menu.rst [hunk 1]
```diff
@@ -138,4 +140,140 @@ After these two steps your user account menu should look like that, the new item
...
+
+**1.** In order to add buttons to the Admin Customer Show menu in **Sylius** you have to create a ``AppBundle\Menu\AdminCustomerShowMenuListener`` class.
+
+.. note::
+
+    **This menu is build from buttons.** There are a few button types available:
+    ``edit``, ``show``, ``delete``, ``link`` (default), and ``transition`` (for state machines).
+
+    Buttons (except for the ``link`` and ``transition`` types) already have a defined color, icon and label.
+    The ``link`` and ``tramsition`` types buttons can be customized with the ``setLabel('label')``, ``setLabelAttribute('color', 'color')``
```
Comment (GSadee) [inline docs/customization/menu.rst]: `tramsition` => `transition`

File: docs/customization/menu.rst [hunk 2]
```diff
@@ -138,4 +140,140 @@ After these two steps your user account menu should look like that, the new item
...
+    # app/config/config.yml
+    imports:
+        - { resource: "services.yml" }
+
+How to customize Admin Order Show Menu?
+---------------------------------------
+
+.. tip::
+
+    Admin order show menu is the set of buttons(currently only `Cancel` button) in the right top corner on the ``/admin/orders/{id}`` url.
```
Comment (GSadee) [inline docs/customization/menu.rst]: space before `(`

File: docs/customization/menu.rst [hunk 3]
```diff
@@ -138,4 +140,140 @@ After these two steps your user account menu should look like that, the new item
...
+
+**1.** In order to add buttons to the Admin Order Show menu in **Sylius** you have to create a ``AppBundle\Menu\AdminOrderShowMenuListener`` class.
+
+.. note::
+
+    **This menu is build from buttons.** There are a few button types available:
+    ``edit``, ``show``, ``delete``, ``link`` (default), and ``transition`` (for state machines).
+
+    Buttons (except for the ``link`` and ``transition`` types) already have a defined color, icon and label.
+    The ``link`` and ``tramsition`` types buttons can be customized with the ``setLabel('label')``, ``setLabelAttribute('color', 'color')``
```
Comment (GSadee) [inline docs/customization/menu.rst]: As above

File: docs/customization/menu.rst [hunk 4]
```diff
@@ -138,4 +140,140 @@ After these two steps your user account menu should look like that, the new item
...
+         */
+        public function addAdminOrderShowMenuItems(MenuBuilderEvent $event)
+        {
+            $menu = $event->getMenu();
+
+            $menu->addChild('new', ['route' => 'sylius_admin_order_index'])
+                ->setAttribute('type', 'link')
+                ->setLabel('Maroon Menu Button')
+                ->setLabelAttribute('icon', 'star')
+                ->setLabelAttribute('color', 'maroon');
```
Comment (GSadee) [inline docs/customization/menu.rst]: `;` to new line

File: docs/customization/menu.rst [hunk 5]
```diff
@@ -138,4 +140,140 @@ After these two steps your user account menu should look like that, the new item
...
+         */
+        public function addAdminCustomerShowMenuItems(MenuBuilderEvent $event)
+        {
+            $menu = $event->getMenu();
+
+            $menu->addChild('new', ['route' => 'sylius_admin_customer_index'])
+                ->setAttribute('type', 'link')
+                ->setLabel('Pink Menu Button')
+                ->setLabelAttribute('icon', 'star')
+                ->setLabelAttribute('color', 'pink');
```
Comment (GSadee) [inline docs/customization/menu.rst]: `;` to new line

**Comment by TheMadeleine** [comment]
Done!

**Comment by GSadee** [comment]
Thanks! 😃

**Pull request merged**